### PR TITLE
Extract command class and rearrange the packages

### PR DESCRIPTION
**Pull request by hlwang56**
Extract all the commands
**PACKAGE HAS CHANGED**

### CODE REVIEW

File: src/main/java/seedu/planitarium/PlanITarium.java [hunk 1]
```diff
@@ -34,8 +36,8 @@ public void run() throws Exception {
         while (true) {
             userInput = new Scanner(System.in);
             logger.getLogger().log(Level.INFO, "Next line has been read");
-            commandExecuter = new Command(userInput.nextLine(), personList);
-            commandExecuter.execute();
+            command = commandFactory.getCommand(userInput.nextLine(), personList);
+            command.execute();
```
Comment (1szheng) [inline src/main/java/seedu/planitarium/PlanITarium.java:40]: Might throw an exception that will be uncaught

File: src/main/java/seedu/planitarium/commands/CommandFactory.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            case EXIT:
+                newCommand = new ExitCommand(userInput, personList);
+                break;
+            default:
+                throw new UnknownException();
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        return newCommand;
```
Comment (1szheng) [inline src/main/java/seedu/planitarium/commands/CommandFactory.java:54]: Default branch for throwing exception will be caught by line 51. Might cause an undefined `newCommand` to be returned by line 54.

File: src/main/java/seedu/planitarium/commands/Command.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    public Command(String userInput, PersonList personList) {
+        this.userInput = userInput;
+        this.personList = personList;
+    }
+
+    /**
+     * Executes the instruction according to the input after parsing.
+     * @throws UnknownException if the instruction cannot be executed.
+    */
+    public void execute() throws Exception {}
```
Comment (1szheng) [inline src/main/java/seedu/planitarium/commands/Command.java:30]: Feel that closing braces should be in new line even if method is empty

**Review by 1szheng — COMMENTED** [review]
Overall good job, some comments for possible issues.

**Comment by hlwang56** [comment]
The part that Si Zheng mention has been fixed.

**Review by 1szheng — APPROVED** [review]
Good job

**Pull request merged**